Source/Extensions/Blazorise.DataGrid/Configuration/VirtualizeState.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Blazorise.DataGrid.Configuration
+{
+    internal struct VirtualizeState
```

Comment:
Better make it a `record` like we already have it in the core project. It is a lot easier to use.

Reply:
I don't see why a record would be better. It's a reference type used for value semantics. Which is not really applicable here? Not necessary to allocate memory? 
Structs still have their use.
Could you elaborate if I'm missing something?

Reply:
It's easier to compare, eg `state1 != state2` since it has all the Equal overloads already implemented. Also, it doesn't copy the entire object each time. I found it really good for our use-cases.

Reply:
Yep value semantics which are currently not needed. And it's internal so we can just change it when it actually makes sense to use the feature in my opinion... But I can change it right away, if you really do make a point in changing it and using records over structures despite both features having different use cases.